chia/rpc/harvester_rpc_api.py

```diff
@@ -23,6 +23,7 @@ def get_routes(self) -> dict[str, Endpoint]:
         return {
             "/get_plots": self.get_plots,
             "/refresh_plots": self.refresh_plots,
+            "/hard_refresh_plots": self.hard_refresh_plots,
```

Comment:
it would seem more intuitive to add an argument to `/refresh_plots`, something like `force=True`, rather than adding a whole new endpoint. Maybe there's a good reason.